examples/ecmascript/es5.ohm

```diff
@@ -338,18 +328,13 @@ ES5 {
                   | "!"    UnaryExpression  -- lnot
                   | PostfixExpression
 
-  MultiplicativeExpression = MultiplicativeExpression "*" UnaryExpression -- mul
-                           | MultiplicativeExpression "/" UnaryExpression -- div
-                           | MultiplicativeExpression "%" UnaryExpression -- mod
+  MultiplicativeExpression = MultiplicativeExpression ("*"|"/"|"%") UnaryExpression -- mul
                            | UnaryExpression
 
-  AdditiveExpression = AdditiveExpression "+" MultiplicativeExpression -- add
-		     | AdditiveExpression "-" MultiplicativeExpression -- sub
-		     | MultiplicativeExpression
+  AdditiveExpression = AdditiveExpression ("+"|"-") MultiplicativeExpression -- add
```

Comment:
We only have to name it `rec` if I get through with my "shorten all the alternatives to a big alternation" :-p.